app/models/patient.js

```diff
@@ -15,6 +15,7 @@ export default AbstractModel.extend(DOBDays, PatientName, {
   bloodType: DS.attr('string'),
   clinic: DS.attr('string'),
   country: DS.attr('string'),
+  createOutpatientVisit: DS.attr('boolean'),
```

Comment:
This attribute should not be here because we do not want to (or need to) save this value to the database.